src/app/app.routes.ts

```diff
@@ -1,3 +1,19 @@
 import { Routes } from '@angular/router';
+import { LoginComponent } from './components/login/login.component';
+import { SignupComponent } from './components/signup/signup.component';
+import { LandingComponent } from './components/landing/landing.component';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {
+    path: '',
+    component: LandingComponent,
+  },
+  {
+    path: 'log-in',
+    component: LoginComponent,
+  },
+  {
+    path: 'sign-up',
+    component: SignupComponent,
+  },
```

Comment:
Don't forget to add an error page and a redirect to the landing component. Also, how are you guarding your components before auth?